core/vm/gas.go

```diff
@@ -173,6 +174,7 @@ var _baseCheck = map[OpCode]req{
 	BLOCKHASH:      {1, GasExtStep, 1},
 	BALANCE:        {1, new(big.Int), 1},
 	EXTCODESIZE:    {1, new(big.Int), 1},
+	EXTCODEHASH:    {1, big.NewInt(400), 1},
```

Comment:
gas should be set by gas table? This does not allow for future proofing since it is a static value.